Minimal Web API/UploadDocument.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace DocumentEditorCore
+{
+    /// <summary>
+    /// Specifies the Document name to upload document
```

Comment:
Represents the parameters required for loading a document that is already stored in the server.